www/gatsby-node.js

```diff
@@ -540,6 +540,13 @@ exports.createPages = ({ graphql, actions, reporter }) => {
         }
       })
 
+      // redirecting cypress-gatsby => gatsby-cypress
+      createRedirect({
```

Comment:
when searching gatsby cypress you get https://www.gatsbyjs.org/packages/cypress-gatsby/ so I've made a redirect as well.